using usando_throw.Models;


new ExemploExcecao().Method1();